pkgs/applications/editors/010-editor/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib, stdenv, fetchFromGitHub, fetchurl, makeDesktopItem, autoPatchelfHook, copyDesktopItems, perlPackages, zlib, fontconfig, freetype, libxkbcommon, cups, ... }:
+
+let
+  extract = fetchFromGitHub {
+    owner = "lod";
+    repo = "unpack-install-jammer";
+    rev = "41faa5a8c7925db4ff7ca665cca478979d177de4";
+    sha256 = "1gvx7nkg4spz245hnmzcshiy0k3qvslqlzp4if7ngdqi2pfjrn4q";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "010-editor";
+  version = "12.0.1";
+
+  src = fetchurl {
+    url = "https://download.sweetscape.com/010EditorLinux64Installer${version}.tar.gz";
+    sha256 = "0ln8qxy1mij8qnnn8dxwnvmkx69l0f0w6xnb615ihx8wyhvlmfzh";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+  ] ++ (with perlPackages; [
+    perl
+    ModernPerl
+    CompressRawLzma
+    CompressRawZlib
+    FileHomeDir
+    TermProgressBar
+    DataDump
+  ]);
+
+  buildInputs = [
+    zlib
+    freetype
+    fontconfig
+    libxkbcommon
+    cups
+  ];
+
+  # Archive does not have a subdirectory
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  # Use shipped Qt5 libs
+  runtimeDependencies = [ "$out" ];
+
+  postUnpack = ''
+    perl ${extract}/extract.pl -s Home=/ 010EditorLinux64Installer
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    cp -ra install_dir/010editor $out
+
+    mkdir $out/bin
+    ln -s $out/010editor $out/bin/010editor
```

Comment:
Why does the binary need to be in /?

Reply:
The other way around. I symlinked it to `$out/bin` to make it end up in `PATH`.

I left the rest of the package untouched, that is, when using the GUI installer it will create the same directory structure. The main executable just doesn't end up in `PATH`.